src/index.html

```diff
@@ -42,29 +42,86 @@
   <body>
     <div id="head">
 
-    <div id="head-left">
-      <h1 id="logo">Summer of Hacks</h1>
+      <div id="head-left">
+        <h1 id="logo">Summer of Hacks</h1>
+      </div>
+
+      <div id="head-right">
+
+        {{#each events}}
+          <h2><a href="#{{slug}}"><span class="date">{{short_date date}}</span> {{name}}</a></h2>
```

Comment:
If we're having all years in the same site, we'll need to add the year as a prefix to the slug (and move the page) so we don't end up with conflicts.